play28-json-tests-sc14/src/main/scala/play/api/libs/json/scalacheck/DurationGenerators.scala

```diff
@@ -0,0 +1,53 @@
+package play.api.libs.json.scalacheck
+
+import org.scalacheck.{Arbitrary, Gen}
+
+import scala.concurrent.duration._
+import scala.concurrent.duration.ops.v4._
+import scala.language.implicitConversions
+
+trait DurationGenerators {
```

Comment:
Use a `class` for better binary compatibility... or only allow importing from the `object`.

Reply:
This would break places that extend this trait for the implicits. This is a problem for v5.